app/structures/weighted_adjacency_list.py

```diff
@@ -161,6 +170,15 @@ class WeightedDirectedAdjacencyList(WeightedAdjacencyList):
     def __init__(self, graph):
         self.graph = graph
 
+    def is_output_vertex(self, vertex, listToCheck = None):
```

Comment:
Why do you suddenly use camelCase variables names?

Reply:
It's force of habit. ;)